core/src/main/scala/cats/data/OneAnd.scala

```diff
@@ -27,7 +27,7 @@ final case class OneAnd[F[_], A](head: A, tail: F[A]) {
    */
   def filter(f: A => Boolean)(implicit F: MonadCombine[F]): F[A] = {
     val rest = F.filter(tail)(f)
-    if (f(head)) F.combineK(F.pure(head), rest) else rest
+    if(f(head)) F.combineK(F.pure(head), rest) else rest
```

Comment:
Later in this diff, spaces are inserted between `if` and `(`, for example [here](https://github.com/typelevel/cats/pull/1142/files#diff-26729163874fb5967d1a0c27b6db2457R60). Can we add a scalastyle rule to make this consistent?